### PR TITLE
audit: add check for indefinite article

### PR DESCRIPTION
**Pull request by afh**
at the beginning of desc.

### CODE REVIEW

File: Library/Homebrew/cmd/audit.rb [hunk 1]
```diff
@@ -367,6 +367,10 @@ def audit_desc
     if desc =~ %r[[Cc]ommand line]
       problem "It should be \"command-line\", not \"command line\"."
     end
+
+    if desc =~ %r[^([Aa]n?)\s]
+      problem "Description begins with indefinite article: \"#{$1}\""
```
Comment (xu-cheng) [inline Library/Homebrew/cmd/audit.rb]: Might be better to offer the correction suggestion, such as `Please remove the indefinite article (\"#{$1}\") in the beginning of description`.

Comment (mikemcquaid) [inline Library/Homebrew/cmd/audit.rb]: Might want to clarify `(i.e. 'a' or 'an')` just for people who don't know what an indefinite article is.

Comment (afh) [inline Library/Homebrew/cmd/audit.rb]: @xu-cheng I was uncertain which wording to use and will happily change it.
 
@mikemcquaid That's what the `#{$1}` is for, so people know which indefinite article was used in the description.

Comment (mikemcquaid) [inline Library/Homebrew/cmd/audit.rb]: Ah, I see.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/4ce9763593aa310cc21a7c610e56aaf376b0679f. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**